examples/jsm/exporters/USDZExporter.js

```diff
@@ -1,5 +1,6 @@
 import * as THREE from 'three';
```

Comment:
Side note: It's not good that `USDZExporter` imports the entire core. Like all other modules, the exporter should only import what is actually required. This should be fixed with another PR.

Reply:
Agreed, I'll plan on submitting a separate PR once/if this is merged.